wgsl/index.bs

```diff
@@ -407,6 +420,11 @@ TODO: This section is a stub.
 
 In [SHORTNAME], a value of [=storable=] type may be stored in memory, for later retrieval.
 
+In general [SHORTNAME] follows the [[!VulkanMemoryModel|Vulkan Memory Model]]
+with the following exceptions
```

Comment:
This is the most important restriction yes. We have followup to pin this down further.